src/components/checkbox/_checkbox.scss

```diff
@@ -318,6 +318,12 @@
   // Disabled
   // ---------------------------------------------
 
+  .#{$prefix}--checkbox:disabled + .#{$prefix}--checkbox-label {
+    [disabled] ~ _ {
```

Comment:
Could we add a note for why we need this, something like:

```
// Bugfix: <issue-link>
```

?

Reply:
Good idea. Done 👍